utils/board.py

```diff
@@ -0,0 +1,92 @@
+from .player import Player
+from .movable import Movable
+from random import randrange
+
+class Board():
+
+    def __init__(self, 
+                board_size, 
+                player_list, 
+                snake_list, 
+                ladder_list,
+                no_of_dice=1):
+
+        self.__board_size = board_size
+        self.__player_list = player_list
+        self.__snake_list = snake_list
+        self.__ladder_list = ladder_list
+
+        self.__no_of_dice = no_of_dice
+
+        self.__curr_player = 0  # current player holding the dice
+        self.__total_players = len(self.__player_list)
+
+        self.__rank = {}
+
+        self.__init_board()
+
+    def __init_board(self):
+        self.__movable_source_to_destination = {}
+
+        for snake in self.__snake_list:
+            self.__movable_source_to_destination[snake.get_source()] = snake.get_target()
+        
+        for ladder in self.__ladder_list:
+            self.__movable_source_to_destination[ladder.get_source()] = ladder.get_target()
+
+    def make_move(self):
+        
+        __player = self.get_current_player()
+        __source = __player.get_current_pos()
+        __dice_val = self.__get_dice_val()
+
+        if not __player in self.__rank:
```

Comment:
will player, source, dice_val be printed even if this is false?

Reply:
yes. ideally it should return. if the player in at 97 and gets 4, it will be still at 97.

Reply:
in that case, this will not return false, right?

Reply:
yes. it will be returning true.